extensions/spring-xd-extension-sqoop/src/main/java/org/springframework/xd/sqoop/SqoopTasklet.java

```diff
@@ -74,6 +86,9 @@ protected boolean isStoppable() {
 				}
 			}
 		}
+		for (Map.Entry e : hadoopProperties.entrySet()) {
+			command.add(SPRING_HADOOP_CONFIG_PREFIX + "." + e.getKey() + "=" + e.getValue());
```

Comment:
That should be pretty easy to fix. However, it is a bit confusing to the end user. Maybe we should switch the properties in the docs to be the corresponding Hadoop properties instead of the Spring ones. It is a hadoop.properties file and not a spring-hadoop.properties file. I was expecting to put Hadoop style properties in this file and not Spring style ones.